shell/platform/windows/text_input_plugin.cc

```diff
@@ -446,8 +446,16 @@ void TextInputPlugin::SendStateUpdateWithDelta(const TextInputModel& model,
 
 void TextInputPlugin::EnterPressed(TextInputModel* model) {
   if (input_type_ == kMultilineInputType) {
+    std::u16string text_before_change = fml::Utf8ToUtf16(model->GetText());
+    TextRange selection_before_change = model->selection();
     model->AddText(std::u16string({u'\n'}));
```

Comment:
Consider just `model->AddText(u"\n");` since the conversion will happen implicitly and it's slightly easier on the eyes :)